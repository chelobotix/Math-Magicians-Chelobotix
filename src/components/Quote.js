import style from '../styles/Quote.module.css';

const Quote = () => (
  <p className={style.quote}>
    &quot;Mathematics is not about numbers, equations, computations
    <br />
    or algorithms: it is about understanding&quot; - William Paul Thurston
  </p>
);

export default Quote;
