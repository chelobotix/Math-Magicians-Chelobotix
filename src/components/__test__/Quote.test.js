import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Quote Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
