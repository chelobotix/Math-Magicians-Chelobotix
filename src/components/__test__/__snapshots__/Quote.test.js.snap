// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component Tree 1`] = `
<p
  className="quote"
>
  "Mathematics is not about numbers, equations, computations
  <br />
  or algorithms: it is about understanding" - William Paul Thurston
</p>
`;
